src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Validation.cs

```diff
@@ -122,18 +124,26 @@ internal static unsafe partial class Utf8Utility
 
                         do
                         {
-                            if (Sse2.IsSupported)
+                            if (AdvSimd.Arm64.IsSupported)
```

Comment:
The comment block beginning on line 137 regarding unaligned reads should be moved before this _if_ block, as that comment applies both to x86 SIMD intrinsics and to ARM64 SIMD intrinsics.

Reply:
Thanks, good observation. I'll move the comment.